firebase-firestore/src/test/java/com/google/firebase/firestore/remote/RemoteSerializerTest.java

```diff
@@ -710,11 +706,10 @@ public void testEncodesDisjunctiveFiltersOnDeeperCollections() {
     assertEquals(serializer.decodeQueryTarget(serializer.encodeQueryTarget(q)), q);
   }
 
-  // PORTING NOTE: Isolated array-contains, array-contains-any, and in filter test omitted since we
-  // seem to have omitted isolated filter tests on Android (and the encodeRelationFilter() /
-  // decodeRelationFilter() serializer methods are private) in favor of relying on the larger tests.
-  // array-contains, array-contains-any, and in encoding / decoding is covered by
-  // testEncodesMultipleFiltersOnDeeperCollections().
+  // PORTING NOTE: Isolated array-contains filter test omitted since we seem to have omitted
+  // isolated filter tests on Android (and the encodeRelationFilter() / decodeRelationFilter()
+  // serializer methods are private) in favor of relying on the larger tests. array-contains
+  // encoding / decoding is covered by testEncodesMultipleFiltersOnDeeperCollections().
```

Comment:
FWIW- In the fullness of time, we should try to bring these tests into parity with TypeScript / iOS which have isolated tests for each filter...  So once you make it convenient to have isolated filter tests perhaps change this to:

> TODO(PORTING NOTE): Android currently tests most filter serialization (for equals, greater than, array-contains, etc.) only in testEncodesMultipleFiltersOnDeeperCollections and lacks isolated filter tests like the other platforms have. We should fix this.